packages/asset-service/src/generateAssetData/avalanche/index.ts

```diff
@@ -1,10 +1,19 @@
 import { avalancheChainId } from '@shapeshiftoss/caip'
 
+import { getRenderedIdenticonBase64 } from '../../service/GenerateAssetIcon'
 import { avax } from '../baseAssets'
 import * as coingecko from '../coingecko'
 
 export const getAssets = async () => {
   const assets = await coingecko.getAssets(avalancheChainId)
   assets.push(avax)
-  return assets
+  return assets.map((asset) => {
+    if (!asset.icon) {
+      asset.icon = getRenderedIdenticonBase64(asset.assetId, asset.symbol, {
+        identiconImage: { size: 128, background: [45, 55, 72, 255] },
```

Comment:
@reallybeard not sure if there is a specific identicon background color meant to be used for a specific chain? ethereum and osmosis seem to use the same background color, so leaving as this unless I hear otherwise

Reply:
We can just roll with what we have for the current assets. We have no specific colors for each chain. You can see the accounts page to see the reference to chain colors.